correctness/README.md

```diff
@@ -1,18 +1,36 @@
 # Correctness Testing for Prometheus Connector
 
 ## Prerequisites
-Prior to running the tests in correctness_test.go, ensure the following:
-1. Have a database called correctness_testing with the table named correctness_testing created.
-2. Ingested data to the correctness_testing table for at least an hour.
-3. Updated the basic_auth section within [correctness_testing.yml](./config/correctness_testing.yml).
-2. Download or build the Prometheus Connector Docker image and store it in a new directory named `resources` in the repository root.
-
-## How to build and save the docker image
-1. Execute the following command to build the docker image:
-`docker buildx build . -t timestream-prometheus-connector-docker`
-2. Execute the following command to save the docker image as a compressed file and update the `version` appropriately:
-`docker save timestream-prometheus-connector-docker | gzip > timestream-prometheus-connector-docker-image-<version>.tar.gz`
-
-## How to execute tests
-1. Run the following command to execute the correctness tests:
-`go test -v ./correctness`
+1. Ensure your AWS credentials are configured for your environment. You can configure it with `aws configure`.
+
+2. Create a new Timestream database and table:
+```
+aws timestream-write create-database --database-name CorrectnessDB --region us-east-1 && aws timestream-write create-table --database-name CorrectnessDB --table-name CorrectnessMetrics
+```
+
+## Run Correctness Tests
+1. Update the default database, table and region in [`docker-compose.yml`](https://github.com/awslabs/amazon-timestream-connector-prometheus/blob/main/docker-compose.yml) with your new Timestream database, table and region.
```

Comment:
It would be great if we didn't have to do any code changes for updating the database and table for correctness tests. Can you update the `docker-compose.yml` to use environment variables with default values that we can override when executing `docker compose`.

Reply:
My preference would be to enable overriding the current `docker-compose.yml`, since overriding defaults is generally useful.